lib/compress/huf_compress.c

```diff
@@ -425,8 +433,8 @@ size_t HUF_buildCTable_wksp (HUF_CElt* tree, const unsigned* count, U32 maxSymbo
  */
 size_t HUF_buildCTable (HUF_CElt* tree, const unsigned* count, unsigned maxSymbolValue, unsigned maxNbBits)
 {
-    huffNodeTable nodeTable;
-    return HUF_buildCTable_wksp(tree, count, maxSymbolValue, maxNbBits, nodeTable, sizeof(nodeTable));
+    U32 workspace[HUF_WORKSPACE_SIZE_U32];
```

Comment:
Since you defined it just above, it would be slightly proper to use `HUF_buildCTable_wksp_tables` type here.
`HUF_WORKSPACE_SIZE_U32` is merely a proxy to get there.